packages/eslint-plugin-wpcalypso/lib/rules/test/i18n-unlocalized-url.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @file Disallow using unlocalized URL strings
+ * @author Automattic
+ * @copyright 2023 Automattic. All rights reserved.
+ * See LICENSE.md file in root directory for full license.
+ */
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const RuleTester = require( 'eslint' ).RuleTester;
+const rule = require( '../i18n-unlocalized-url' );
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const localizableUrls = [
+	'https://apps.wordpress.com/',
+	'https://automattic.com/cookies/',
+	'https://automattic.com/privacy/',
+	'https://wordpress.com/support/test-article/',
+	'https://wordpress.com/forums/',
+	'https://wordpress.com/tos/',
+];
+
+const nonLocalizableUrls = [ 'https://example.com', 'https://wordpress.com/forums/some-thread' ];
```

Comment:
Analogically to the localizable URLs, perhaps we can add a few more scenarios for the URLs that don't expect further paths, like `https://automattic.com/privacy`.

Reply:
Added some more in https://github.com/Automattic/wp-calypso/pull/78480/commits/2489f7a3be20ad3d3c8533b028a257e0972d013c.